apps/react-18-tests-v8/jest.config.js

```diff
@@ -11,14 +11,6 @@ const config = createConfig({
   snapshotSerializers: ['@fluentui/jest-serializer-merge-styles'],
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32280&commitId=bea9356e2313e31c067bc873c555b0331924d8cd&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|
|Avatar Converged.badgeMask.chromium.png|36| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|3315| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>